pmm-app/src/pmm-add-instance/components/Discovery/components/Credentials/Credentials.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { stylesFactory } from '@grafana/ui';
+import { css } from 'emotion';
+
+export const getStyles = stylesFactory(() => ({
+  instanceForm: css`
+    padding: 0px;
+    margin-bottom: 10px;
```

Comment:
Can use spacing here from the theme and below on margin-top too

Reply:
replaced it here, 2px margin in this file was added because height of the input field and button are different and it was the fastest way to align it together. I wrote it down to fix this issue later